seed/seedwriter/writer.go

```diff
@@ -199,6 +207,8 @@ type Writer struct {
 	systemSnap                      *SeedSnap
 	kernelSnap                      *SeedSnap
 	noKernelSnap                    bool
+	manifest                        *Manifest
```

Comment:
as discussed, looks like duplication of options

Reply:
Nice catch, reused ManifestPath from options instead. I do keep a seperate 'copy' of the Manifest as the manifest may be initialized to a new instance.